discrete_env.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+from gym import Env, spaces
+from gym.utils import seeding
+
+
+def categorical_sample(prob_n, np_random):
+    """
+    Sample from categorical distribution
+    Each row specifies class probabilities
+    """
```

Comment:
Please follow the [docstring format in FastForest](https://github.com/ThrunGroup/FastForest/blob/main/experiments/exp_utils.py#L12-L18) (with parameters and return)